packages/create-plugin/src/constants.ts

```diff
@@ -32,13 +33,14 @@ export enum PLUGIN_TYPES {
   panel = 'panel',
   datasource = 'datasource',
   secretsmanager = 'secretsmanager',
+  scenes = 'scenesapp',
 }
 
 // This gets merged into variables coming from user prompts (when using Plop) or any other dynamic variables,
 // and will be available to use in the templates.
 // Example: "@grafana/ui": "{{ grafanaVersion }}"
 export const EXTRA_TEMPLATE_VARIABLES = {
-  grafanaVersion: '9.3.8',
+  grafanaVersion: '9.5.0',
```

Comment:
@dprokop any reason not to go to 9.5.2?

Reply:
@sympatheticmoose primarily because of https://github.com/grafana/plugin-tools/pull/246 but I think we should get 9.5.2 there too, WDYT @jackw?

Reply:
ah gotcha, hadn't seen it was also set in that PR 👍

Reply:
FYI - I bumped #246 to 9.5.2 prior to merging. Well spotted! 🕵️